rest_api.py

```diff
@@ -101,10 +103,22 @@ def get(self):
     return json.dumps(response)
 
 
+class AirflowDB(Resource):
```

Comment:
I'm not entirely sure how to do this, some suggestions would be really helpful

Reply:
This class should probably do connection and result set management.
i.e. this owns the connection(s) and close them when server exits, instead of using a global DB connection.

You may also want to verify that you can open multiple cursors on the same connection.. I remember some old jdbc drivers support only one at a time.

Reply:
If we are associating the DB connection with the class, then the server would have to open a new connection each time a request is made if we put it in a 'init' function which seems to me as if it negates the idea of having a consistent connection open.

Additionally, it seems that while my brief testing seems to allow for multiple cursors to be open on the same time, the documentation suggests closing the cursor after executing a query, which I have added

Reply:
DB connection should have proper scope. In your case you can open it when server starts and close it when server stops, but it (along with most things in any server) should not be global objects.